packages/unlock-js/src/unlockService.ts

```diff
@@ -139,7 +139,9 @@ export default class UnlockService {
       const contractVersion = await contract.publicLockVersion()
       version = parseInt(contractVersion, 10) || 0
     } catch (error) {
-      // This is an older version of Unlock which did not support publicLockVersion
+      console.error(
+        `We could not retrieve the version of the Unlock contract ${address} on this network.`
```

Comment:
what is the case where we could not retrieve the lock version? All locks have this, no ?

Reply:
Generally I think it would be a misconfiguration... and I bumped into that yesterday :) Adding the error makes it not silent!